change/@internal-react-composites-7c1614bf-ee32-4f62-bc78-2ba07a5aa3a9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "this change does not affect the published package in any way.",
```

Comment:
This actually does impact the composites:
![image](https://user-images.githubusercontent.com/2684369/134053267-8c8f9095-671e-43be-b61d-7e2e3acec559.png)

https://github.com/Azure/communication-ui-library/pull/789/files?short_path=710bcfd#diff-710bcfd462fa4c1aa7e00a93f85f0320b9b9761ad40ea523fcd25ae90fd891ec

Is the difference padding intentional / meet figma specs?

Reply:
Thanks for catching that. I have added margins to fix this. But now the font weight is no longer 600 and the icon and label are tighter together which is closer to the Figma spec.